api/api_storage.go

```diff
@@ -102,6 +104,9 @@ type StorageMiner interface {
 	PiecesGetPieceInfo(ctx context.Context, pieceCid cid.Cid) (*piecestore.PieceInfo, error)
 	PiecesGetCIDInfo(ctx context.Context, payloadCid cid.Cid) (*piecestore.CIDInfo, error)
 
+	// MinerRestartDataTransfer attempts to restart a data transfer with the given transfer ID and other peer
+	MinerRestartDataTransfer(ctx context.Context, transferID datatransfer.TransferID, otherPeer peer.ID, isInitiator bool) error
```

Comment:
Should probably be prefixed with `Markets` and be next to `MarketDataTransferUpdates`

Reply:
agree